java/java.source/nbproject/project.xml

```diff
@@ -222,6 +222,15 @@
                         <specification-version>1.58</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
+                    <code-name-base>org.netbeans.modules.extexecution</code-name-base>
```

Comment:
Such dependency makes the `java.source` module more [heavyweight](http://wiki.apidesign.org/wiki/HeavyWeight). Is that OK, @jlahoda?

Reply:
The reason why I was doing this in the first place, is that I was using NbPreferences to store arguments, and VM Options. And NbPreferences require for the accessing class to be in the same module, but now, I am using dataobject's attribute, so I can move this class out of java.source.